### ISSUE TITLE
Why use BCELoss instead Cross Entropy?

### ISSUE THREAD

**Issue by Edwardmark** [ultralytics/yolov5#4064]
## ❔Question
 Why use BCELoss instead Cross Entropy? Is there any specific reason for that?

**Comment by glenn-jocher** [comment]
Why use CE?

**Comment by Edwardmark** [comment]
> Why use CE?

sorry to ask suck a silly question, but when do classification, softmax loss is often used, I just want to know, why in yolo, we use BCEloss instead of softmax loss? Thank you very much.

**Comment by glenn-jocher** [comment]
@Edwardmark see https://arxiv.org/abs/1804.02767

**Comment by Edwardmark** [comment]
@glenn-jocher 
I see, it is because that the label may be overlapping, thanks.
In yolo-v3, it says:
```
During training we use binary cross-entropy loss for the class
predictions.
This formulation helps when we move to more complex
domains like the Open Images Dataset [7]. In this dataset
there are many overlapping labels (i.e. Woman and Person).
Using a softmax imposes the assumption that each box has
exactly one class which is often not the case. A multilabel
approach better models the data.
```
